forms.rst

```diff
@@ -389,349 +519,363 @@ object.
         }
 
 That's it! If you re-submit the form with invalid data, you'll see the
-corresponding errors printed out with the form.
+corresponding errors printed out with the form. Read the
+:doc:`Symfony validation documentation </validation>` to learn more about this
+powerful feature.
 
-Validation is a very powerful feature of Symfony and has its own
-:doc:`dedicated article </validation>`.
+Other Common Form Features
+--------------------------
 
-.. _forms-html5-validation-disable:
+Field Type Options
```

Comment:
```suggestion
Form Type Options
```

To be consistent. Also, IIRC Symfony used to have "form" and "field" types in Symfony 2.0, so this might be confusing to people.